skills/morph-ppt/reference/morph-helpers.sh

```
@@ -0,0 +1,361 @@
+    echo -e "${BLUE}📋 Cloning slide $from_slide → $to_slide...${NC}"
+    officecli add "$deck" '/' --from "/slide[$from_slide]"
+
+    echo -e "${BLUE}⚡ Setting morph transition...${NC}"
+    officecli set "$deck" "/slide[$to_slide]" --prop transition=morph
+
+    echo -e "${BLUE}📊 Listing shapes for ghosting reference:${NC}"
+    officecli get "$deck" "/slide[$to_slide]" --depth 1
+
+    # Verify transition was set
+    echo -e "${BLUE}🔍 Verifying transition...${NC}"
+    local trans=$(officecli get "$deck" "/slide[$to_slide]" --json 2>/dev/null | grep '"transition": "morph"')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)